pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -600,6 +601,7 @@ func ValidateValidatingAdmissionPolicy(p *admissionregistration.ValidatingAdmiss
 func validateValidatingAdmissionPolicy(p *admissionregistration.ValidatingAdmissionPolicy) field.ErrorList {
 	allErrors := genericvalidation.ValidateObjectMeta(&p.ObjectMeta, false, genericvalidation.NameIsDNSSubdomain, field.NewPath("metadata"))
 	allErrors = append(allErrors, validateValidatingAdmissionPolicySpec(p.ObjectMeta, &p.Spec, field.NewPath("spec"))...)
+	allErrors = append(allErrors, validateValidatingAdmissionPolicyStatus(&p.Status, field.NewPath("status"))...)
```

Comment:
Just noticed this was adding to ValidateValidatingAdmissionPolicy

Validating both spec and status in the same function is not ideal, since no endpoint can simultaneously update both of those stanzas if we somehow get in a state where those are unhappy with an existing object for some reason.

I'd recommend this instead:

```go
func ValidateValidatingAdmissionPolicyStatusUpdate(p *admissionregistration.ValidatingAdmissionPolicy) field.ErrorList {
  return validateValidatingAdmissionPolicyStatus(&p.Status, field.NewPath("status"))
}
```

called from the status `validatingAdmissionPolicyStatusStrategy.ValidateUpdate` method

Reply:
that also saves you lots of cost validating CEL stuff in spec for a status update which couldn't change anything in those fields anyway

Reply:
Update. Thank you.